testcode/lib/atari/displaylist.c

```diff
@@ -0,0 +1,69 @@
+/*
+** testprogram for ANTIC instructions as defined in "_antic.h"
+**
+** 23-Feb-2017, Christian Krueger
+*/
+
+#include <conio.h>
+#include <atari.h>
+#include <peekpoke.h>
+#include <string.h>
+
+// code is only for testing purposes, as screen and display list are not aligned
+// and jumps not set!
+
+unsigned char DummyScreen[400];
+
+void DisplayList = {
```

Comment:
What is this? Is this C?

What's wrong with
`unsigned char DisplayList[] = { ... };` ?